app/assets/javascripts/discourse/app/lib/ajax-error.js

```diff
@@ -71,5 +85,11 @@ export function throwAjaxError(undoCallback) {
 
 export function popupAjaxError(error) {
   const dialog = getOwner(this).lookup("service:dialog");
-  dialog.alert(extractError(error));
+  const errorInfo = extractErrorInfo(error, undefined, true);
```

Comment:
Minor, but `extractErrorInfo` above only takes two parameters, so the `true` parameter seems redundant.

Reply:
Oops, I was originally going to change `extractError` and that change slipped through.